src/js/parser.js

```diff
@@ -361,16 +373,11 @@ var codeMirrorFn = function() {
             if (ch === '(' && state.tokenIndex !== -4) { // tokenIndex -4 indicates message command
                 stream.next();
                 state.commentLevel++;
-            } else if (ch === ')') {
```

Comment:
Just a note to self: this is a bit that just gobbles up all right-parentheses without so much as dropping an error if something's amiss.